### PR TITLE
Added details about the `organization` parameter

### PR DESCRIPTION
**Pull request by kegashe**
Added details about the `organization` parameter as described here:
- [DOCS-3038](https://auth0team.atlassian.net/browse/DOCS-3038)
- [IUM-1472](https://auth0team.atlassian.net/browse/IUM-1472)

[DOCS-3038]: https://auth0team.atlassian.net/browse/DOCS-3038?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IUM-1472]: https://auth0team.atlassian.net/browse/IUM-1472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: articles/api/authentication/_change-password.md [hunk 1]
```diff
@@ -52,9 +54,11 @@ curl --request POST \
 "We've just sent you an email to reset your password."
 ```
 
-Given a user's `email` address and a `connection`, Auth0 will send a change password email.
+Send a change password email to the user with the given email address on the given connection.
```
Comment (avanscoy) [inline articles/api/authentication/_change-password.md:58]: Suggestion: Send a change password email to the user's provided email and `connection`.

File: articles/api/authentication/_change-password.md [hunk 2]
```diff
@@ -52,9 +54,11 @@ curl --request POST \
...
 
-This endpoint only works for database connections.
+Optionally, you may provide an Organization ID to support Organization-specific variables in customized email templates and redirect URLs.
+
```
Comment (avanscoy) [inline articles/api/authentication/_change-password.md:60]: ^ is there any Org doc we could provide them with the variables or params? This is a good edit! If I was reading it tho, I would want to know where I find those. Just a suggestion tho.

File: articles/api/authentication/_change-password.md [hunk 3]
```diff
@@ -63,6 +67,7 @@ This endpoint only works for database connections.
 | `client_id` | The `client_id` of your client. We strongly recommend including a Client ID so that the email template knows from which client the request was triggered. |
 | `email` <br/><span class="label label-danger">Required</span> | The user's email address. |
 | `connection` <br/><span class="label label-danger">Required</span> | The name of the database connection configured to your client. |
+| `organization` | The `organization_id` of the Organization the user is a member of. |
```
Comment (avanscoy) [inline articles/api/authentication/_change-password.md]: Just a little awkward syntax here:
The `organization_id` of the Organization associated with the user.

File: articles/api/authentication/_change-password.md (hunk 1 above)
Comment (kegashe) [reply articles/api/authentication/_change-password.md:58]: ☝️ Better

File: articles/api/authentication/_change-password.md (hunk 2 above)
Comment (kegashe) [reply articles/api/authentication/_change-password.md:60]: Added a link to email template variables. Checked the MGMT API and included more info regarding redirect URLs behavior.

**Review by avanscoy — APPROVED** [review]

**Comment by avanscoy** [comment]
Great job!

**Review by pavokta — APPROVED** [review]

**Pull request merged**